bin/sosetup

```diff
@@ -2384,7 +2384,9 @@ if [ "$OUTPUT" = "gui" ]; then
 	zenity --progress --title="$TITLE" --text="Please wait while making changes to the system..." --percentage=0 --auto-close
 else
 	echo "Please wait while..."
-	CONFIGURE_SERVICES | grep --line-buffered "^#" | sed 's|# Please wait while||g'
+	set -eo pipefail
+	CONFIGURE_SERVICES | grep --line-buffered "^#" || true | sed 's|# Please wait while||g' || true
+	set +eo pipefail
```

Comment:
Hmm...instead of adding 2 `|| true` to the `CONFIGURE_SERVICES` line, could we instead move `set -eo pipefail` inside the `CONFIGURE_SERVICES` function at the very beginning and then move `set +eo pipefail` to the very end of `CONFIGURE_SERVICES`?